packages/helper-grammar-regex-collection/index.js

```diff
@@ -1,6 +1,6 @@
 import go from './go.js';
 
-const REQUIRE = /require(?:\.resolve)?(?:\s|\()['"]([^'"\s]+)['"]\)?/g;
+const REQUIRE = /require(?:\.resolve)?(?:\s|\()(?: +)?['"]([^'"\s]+)['"](?: +)?\)?/g;
```

Comment:
I think this should be changed to

```js
const REQUIRE = /require(?:\.resolve)?(?:\s|\()\s*['"]([^'"\s]+)['"]\s*\)?/g;
```

Using `\s*` instead of `(?: +)?` is shorter and it will match other whitespace characters like tabs. We should also add some tests with tab characters.